Expressions/constant_expressions.md

```diff
@@ -1,11 +1,21 @@
-# Constant expressions
+# [상수 식(Constant expressions)](#constant-expressions)
+
+* Go 버전: 1.9
+* 원문: [Constant expressions](https://golang.org/ref/spec#Constant_expressions)
+* 번역자: Sang-joon, Moon(@dakeshi)
 
 Constant expressions may contain only [constant](/Constants/) operands and are evaluated at compile time.
 
+상수 식(expression)은 컴파일 시 평가되며, [상수](/Constants/)로만 피연산자들을 구성할 수도 있다.
+
 Untyped boolean, numeric, and string constants may be used as operands wherever it is legal to use an operand of boolean, numeric, or string type, respectively. Except for shift operations, if the operands of a binary operation are different kinds of untyped constants, the operation and, for non-boolean operations, the result use the kind that appears later in this list: integer, rune, floating-point, complex. For example, an untyped integer constant divided by an untyped complex constant yields an untyped complex constant.
 
+불리언, 숫자, string 타입의 피연산자를 사용할 수 있는 곳이라면, 미지정 타입의 불리언, 숫자, 문자열 상수를 피연산자로 사용할 수도 있다. 시프트 연산을 제외하고 이항 연산자의 피연산자가 서로 다른 미지정 타입의 상수일 때, 연산(불리언 연산은 제외) 및 결과의 타입은 다음과 같은 순서로 결정된다: 복소수 상수가 가장 우선되며, 부동 소수점, 룬, 정수 상수 순이다. 예를 들어, 미지정 타입의 정수 상수를 미지정 타입의 복소수 상수로 나눈 결과는 미지정 타입의 복소수 상수다.
+
 A constant [comparison](/Expressions/comparison_operators.html) always yields an untyped boolean constant. If the left operand of a constant [shift expression](/Expressions/operators.html) is an untyped constant, the result is an integer constant; otherwise it is a constant of the same type as the left operand, which must be of [integer type](/Types/numeric_types.html). Applying all other operators to untyped constants results in an untyped constant of the same kind (that is, a boolean, integer, floating-point, complex, or string constant).
 
+상수들 간의 [비교](/Expressions/comparison_operators.html) 결과는 항상 미지정 타입의 불리언 상수다. 왼쪽 피연산자가 미지정 타입의 상수인 상수 [시프트 식](/Expressions/operators.html)의 결과는 정수 상수다; 왼쪽 피연산자가 미지정 타입의 상수가 아닌 경우도 결과는 [정수](/Types/numeric_types.html) 상수다. 이외의 연산자들을 미지정 타입의 상수와 함께 사용하면, 그 결과는 같은 종류의 미지정 상수다. (즉, 불리언, 정수, 부동 소수점, 복소수, 문자열 상수)
```

Comment:
상수 [시프트 식](/Expressions/operators.html)의 왼쪽 피연산자가 미지정 타입의 상수이면 그 연산 결과는 정수 상수이다; 그 외의 경우는 왼쪽 피연산자가 [정수 타입](/Types/numeric_types.html) 상수인 경우인데 연산 결과는 같은 타입의 상수가 된다.

Reply:
> 상수 시프트 식의 왼쪽 피연산자가 미지정 타입의 상수이면 그 연산 결과는 정수 상수다;  

깔끔해졌네요. 이 문장은 제안해주신 내용대로 반영하겠습니다. 👍 

> 왼쪽 피연산자가 미지정 타입의 상수가 아닌 경우도 결과는 [정수](/Types/numeric_types.html) 상수다.

otherwise때문에 고민하던 부분이었습니다. 왼쪽 피연산자는 정수 타입 상수만 허용되므로 `왼쪽 피연산자가 미지정 타입의 상수가 아닌 경우` 보다는 `그 외의 경우는 왼쪽 피연산자가 정수 타입 상수인 경우인데` 라는 표현이 보다 명확할 것 같네요. integer constant 는 `정수 상수`로 번역하겠습니다.

> 그 외의 경우는 왼쪽 피연산자가 정수 상수인 경우인데 연산 결과는 정수 상수다.

Reply:
👍